### PR TITLE
Fix/libp2p connection times

### PR DESCRIPTION
**Pull request by EmiM**
- [x] Add tests (waiting for fixes in base branch)

### CODE REVIEW

File: packages/backend/src/libp2p/websocketOverTor/index.ts [hunk 1]
```diff
@@ -169,9 +169,8 @@ class WebSockets extends EventEmitter {
     const abort = new Promise((_resolve, reject) => {
       onAbort = () => {
         reject(new AbortError())
-        // FIXME: https://github.com/libp2p/js-libp2p-websockets/issues/121
-        setTimeout(() => {
-          rawSocket.close()
+        rawSocket.close().catch(err => {
+          log.error('error closing raw socket', err)
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/backend/src/libp2p/websocketOverTor/index.ts:173]: Tbh. I don't know. I just adjusted this part to match the libp2p Websockets transport.

File: packages/backend/src/storage/localDB.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      return null
+    }
+  }
+
+  public async initPeersStats(peers: string[]) {
+    // Save info about existing peers
+    // Do we care about them?
+    const batchOperations = []
+    for (const peerAddress of peers) {
+      batchOperations.push({ type: 'put', sublevel: this.peers, key: peerAddress, value: {} })
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/backend/src/storage/localDB.ts:70]: I was thinking to not using that method at all, I don't think we need that.

File: packages/backend/src/libp2p/connectionsManager.ts [hunk 3]
```diff
@@ -708,10 +732,7 @@ export class ConnectionsManager extends EventEmitter {
         connectionEncryption: [noise()],
         peerDiscovery: [
           bootstrap({
-            list: params.bootstrapMultiaddrsList,
-            timeout: 120_000, // in ms,
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/backend/src/libp2p/connectionsManager.ts:712]: Yes, 1s so it basically starts discovering peers right away.

[the remark this answers is not in the document]
Comment (vinkabuki) [reply packages/backend/src/libp2p/connectionsManager.ts:712]: great

**Review by siepra — APPROVED** [review]
Did you check if it works on mobile? I mean if there's no need to ship some additional binaries for level?

**Review by vinkabuki — APPROVED** [review]

**Pull request merged**